### ISSUE TITLE
minikube command 'version' tries to issue an http call to https://storage.googleapis.com/minikube/releases.json

### ISSUE THREAD

**Issue by secat** [kubernetes/minikube#408]
With minikube 0.7.0 on Windows platform when we try the `minikube version` command, we end up with the following error log output:

```
$ minikube version
E0727 06:15:38.795354   51816 notify.go:54] Get https://storage.googleapis.com/minikube/releases.json: dial tcp 172.217.2.144:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.
minikube version: v0.7.0
```

However, when I set the shell variable `http_proxy` and `https_proxy` (since I am behind a corporate firewall), the error log message does not appear.

IMHO, the `version` command should not attempt any connection at all and only output the minikube version.

**Comment by jimmidyson** [comment]
Looks like it's trying to check for updated version. I agree version subcommand shouldn't do that. Fancy submitting a PR to fix that?

**Comment by jimmidyson** [comment]
Fix in #412
